recipes-bsp/firmware-imx/firmware-nxp-wifi_1.0.bb

```diff
@@ -80,7 +80,6 @@ PACKAGES =+ " \
 FILES:${PN}-bcm4359-pcie = " \
     ${nonarch_base_libdir}/firmware/brcm/brcmfmac4359-pcie.* \
     ${sysconfdir}/firmware/BCM4349B1_*.hcd"
-RPROVIDES:${PN}-bcm4359-pcie = "linux-firmware-bcm4359-pcie"
```

Comment:
I don't understand the reason for dropping it (the one provided in the commit log) Could you please elaborate a bit?

Is it only a matter of it being replaced?

Reply:
It was Otavio that had specific design reasons not to make an alias from this recipe to the `linux-firmware` packages.

@otavio, can you give me the reasoning as you see it and I'll add it to the commit message?

Reply:
Sure. This package doesn't aim to be a replacement for `linux-firmware` but to offer different firmware files. Forcing user to use the `firmware-nxp-wifi` allow for easier find for them if we eventually have a replacement in `linux-firmware`.